google-cloud-asset-v1beta1/README.md

```diff
@@ -19,7 +19,7 @@ In order to use this library, you first need to go through the following steps:
 1. [Select or create a Cloud Platform project.](https://console.cloud.google.com/project)
 1. [Enable billing for your project.](https://cloud.google.com/billing/docs/how-to/modify-project#enable_billing_for_a_project)
 1. [Enable the API.](https://console.cloud.google.com/apis/library/cloudasset.googleapis.com)
-1. {file:AUTHENTICATION.md Set up authentication.}
```

Comment:
This library is a special case. It's discontinued (and we probably should delete it, but I still need to figure out how to do so safely), so leave this file as is.